Client/Ecosia/Migration/WelcomeScreen.swift

```diff
@@ -0,0 +1,99 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+import UIKit
+import Core
+
+final class WelcomeScreen: UIViewController {
```

Comment:
We now have `Welcome` and `WelcomeScreen` which is a bit unlucky regarding naming imho. As Welcome is the general Welcome for all new installs, I would suggest renaming this one. As it's for people that know the app and update I could imagine something like:
- UpdateScreen, UpdateInfoScreen, UpdateWelcomeScreen, WhatsNewScreen or similar 🤔